[
    {
        "content": "<p>Does anyone understand where the following error comes from?<br>\n<a href=\"https://github.com/leanprover-community/mathlib/runs/6729106015?check_suite_focus=true\">https://github.com/leanprover-community/mathlib/runs/6729106015?check_suite_focus=true</a></p>",
        "id": 284901163,
        "sender_full_name": "Adam Topaz",
        "timestamp": 1654273423
    },
    {
        "content": "<p>ping <span class=\"user-mention\" data-user-id=\"488744\">@Jack M</span></p>",
        "id": 284901179,
        "sender_full_name": "Adam Topaz",
        "timestamp": 1654273430
    },
    {
        "content": "<p>I suspect it's some issue with <span class=\"user-mention\" data-user-id=\"488744\">@Jack M</span> 's git setup, but we can't figure out what the issue is.</p>",
        "id": 284901280,
        "sender_full_name": "Adam Topaz",
        "timestamp": 1654273471
    },
    {
        "content": "<p>The PR  <a href=\"https://github.com/leanprover-community/mathlib/pull/14542\">#14542</a> has the same error. It seems that both <span class=\"user-mention\" data-user-id=\"488744\">@Jack M</span> and <span class=\"user-mention\" data-user-id=\"488747\">@Michael Blyth</span> have some issue with their setup. Does anyone have any ideas?</p>",
        "id": 284905891,
        "sender_full_name": "Adam Topaz",
        "timestamp": 1654275794
    },
    {
        "content": "<p>It looks like some impurity from the runner machine to me, i.e. not your fault</p>",
        "id": 284907981,
        "sender_full_name": "Sebastian Ullrich",
        "timestamp": 1654276906
    },
    {
        "content": "<p>I mentioned this in the maintainer stream.</p>",
        "id": 284908128,
        "sender_full_name": "Adam Topaz",
        "timestamp": 1654276965
    },
    {
        "content": "<p>On lines 309-310 the log says</p>\n<div class=\"codehilite\" data-code-language=\"Text only\"><pre><span></span><code>  /usr/bin/git reset --hard HEAD\n  HEAD is now at d60657c3e Added the concept of a line in projective space.\n</code></pre></div>\n<p>which is the commit with the submodule</p>",
        "id": 284912063,
        "sender_full_name": "Reid Barton",
        "timestamp": 1654278971
    },
    {
        "content": "<p>I suspect that whatever runner tried to build that commit with a submodule is now stuck</p>",
        "id": 284913121,
        "sender_full_name": "Reid Barton",
        "timestamp": 1654279502
    },
    {
        "content": "<p>Aha, so the current CI run on master made it past the checkout step, which is promising.<br>\n<a href=\"https://github.com/leanprover-community/mathlib/runs/6730598604?check_suite_focus=true\">https://github.com/leanprover-community/mathlib/runs/6730598604?check_suite_focus=true</a></p>\n<p>It does indeed look like the issue is with a single runner.</p>",
        "id": 284914729,
        "sender_full_name": "Adam Topaz",
        "timestamp": 1654280299
    },
    {
        "content": "<p>Did somebody do something to fix the error? It looks like nael is fine again.</p>",
        "id": 284918240,
        "sender_full_name": "Gabriel Ebner",
        "timestamp": 1654281851
    },
    {
        "content": "<p>Oh yes, hello, I am fine!</p>",
        "id": 284918360,
        "sender_full_name": "Yaël Dillies",
        "timestamp": 1654281921
    },
    {
        "content": "<p>I was talking about <em>n</em>ael the VM, not Yaël the person. <span aria-label=\"smile\" class=\"emoji emoji-1f642\" role=\"img\" title=\"smile\">:smile:</span></p>",
        "id": 284918629,
        "sender_full_name": "Gabriel Ebner",
        "timestamp": 1654282074
    },
    {
        "content": "<p>The only thing I did was delete the (empty) folder from same branch as the commit that Reid mentioned and pushed to that same branch</p>",
        "id": 284918791,
        "sender_full_name": "Adam Topaz",
        "timestamp": 1654282163
    },
    {
        "content": "<p><a href=\"https://github.com/leanprover-community/mathlib/tree/projective_linear_subspace\">branch#projective_linear_subspace</a></p>",
        "id": 284918873,
        "sender_full_name": "Adam Topaz",
        "timestamp": 1654282207
    }
]